CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+* add compatibility to Symfony v7 and PHP 8.3
+* drop compatibility to Symfony v4 and PHP 7.4
```

Comment:
we shouldn't label it as braking?

Reply:
I don't see mentions of BC breaks in our changelog.
I guess we can simply publish the new release as a major version (v3.0.0), the breaking is implicit

Reply:
👍